scripts/bert/finetune_classifier.py

```diff
@@ -698,8 +743,9 @@ def evaluate(loader_dev, metric, segment):
                         num_calib_batches,
                         quantized_dtype,
                         calib_mode)
-        except AttributeError:
+        except AttributeError as e:
+            warnings.warn(e)
             nlp.utils.version.check_version('1.7.0', warning_only=True, library=mx)
-            warnings.warn('INT8 Quantization for BERT need mxnet-mkl >= 1.6.0b20200115')
+            warnings.warn('INT8 Quantization for BERT need mxnet-mkl >= 1.6.0b20200115 or mxnet >= 1.7')
```

Comment:
since the version check checks for 1.7, there's no need to keep the 1.6 nightly version in the warning.